packages/playground/compile-wordpress/Dockerfile

```diff
@@ -129,15 +174,44 @@ RUN ls -la && \
     exit 'WordPress installation failed'; \
     fi
 
+# === Run WordPress Installer  for the src install ===
+#RUN if [[ -d wordpress-develop-trunk ]] ; then \
```

Comment:
Is this code path used in `wordpress-develop` build? If so, https://github.com/WordPress/wordpress-playground/pull/161/files#r1152126565 it could likely be merged with the RUN above into a single `RUN`. Again, I'm happy to play with it.